Documentation/not-supported-mod-list-and-reasons-why.md

```diff
@@ -2,10 +2,12 @@
 
 |Name							|Version			|Why not working						|What was made for that
 |----							|-------			|---------------						|----------------------
-|Blue Shift						|The latest steam release	|Uses vgui2 library which xash3d does not support		|Recreated source code here: https://github.com/FWGS/hlsdk-xash3d/tree/bshift
-|Counter Strike						|1.6(The latest steam release)	|Uses vgui2 library which xash3d does not support		|Some work on vgui2 support was made here: https://github.com/FWGS/xash3d/tree/vinterface
-|Counter Strike: Condition Zero				|The latest steam release	|Uses vgui2 library which xash3d does not support		|Some work on vgui2 support was made here: https://github.com/FWGS/xash3d/tree/vinterface
-|Counter Strike: Condition Zero - Deleted scenes	|The latest steam release	|Uses vgui2 library which xash3d does not support		|Some work on vgui2 support was made here: https://github.com/FWGS/xash3d/tree/vinterface
-|Day of Defeat						|The latest steam release	|Uses vgui2 library which xash3d does not support		|Some work on vgui2 support was made here: https://github.com/FWGS/xash3d/tree/vinterface
-|Sven-Coop						|5.0+				|Uses custom GoldSrc engine					|
-
+|Arrange Mod: Rebirth					|v150				|								|
```

Comment:
No comments about it though?

Reply:
I'm not sure why.
Qwertyus's comment: too hardly modified.
But mod based on Spirit on Half-Life anyway.

Reply:
I mean, the doc is called "not supported mods and reasons why", and there is no documented reason to not support it.

Maybe put "we don't know yet why it doesn't work"?

Reply:
not-supported-mod-list-and-we-dont-know-why